edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -1274,13 +1286,65 @@ fn get_notary_parameters(
     Some((notary_auth, gun, tag, config_path.to_path_buf()))
 }
 
+// Disallow adding privileged and other capabilities if allow_privileged_docker_containers is false
+fn unset_privileged(
```

Comment:
Can you make this do an early return like `drop_unsafe_privileges` below? That is easier to read than the combined conditional.

```rust
fn unset_privileges() {
    if allow_privileged_docker_containers {
        return;
    }
    if let Some(config) = create_options.host_config() {
        if config.privileged() == Some(&true) || !config.cap_add().is_empty() {
            ...
```